Cargo.toml

```diff
@@ -44,7 +44,8 @@ time = { workspace = true, optional = true, features = [
     "std",
 ] }
 zeroize = { version = "1.8.1", optional = true, features = ["zeroize_derive"] }
-zstd = { version = "0.13.1", optional = true, default-features = false }
+# zstd = { version = "0.13.1", optional = true, default-features = false }
+zstd = { git = "https://github.com/gyscos/zstd-rs", rev = "a3738d680542e34b5529207ee30491ed7b69ed71", optional = true, default-features = false }
```

Comment:
This is to take advantage of https://github.com/gyscos/zstd-rs#288, which is necessary for `zstd` support for this approach. When `zstd-rs` releases a new version, we will be able to depend upon that new release.

Reply:
Version `0.13.2` has been released; can we use it?

Reply:
Yes! Done :D